### PR TITLE
sage.{finance,interacts,libs,numerical,stats,tests}: Replace imports from sage.*.all for namespace packages

### LINKED ISSUE #34957: sage.{finance,interacts,libs,numerical,stats,tests}: Replace imports from sage.*.all for namespace packages
<div id="comment:0"></div>

Part of meta-ticket #34201

CC:  @alexchandler100

Component: **refactoring**

Author: **Alex Chandler**

_Issue created by migration from https://trac.sagemath.org/ticket/34957_

### PR DESCRIPTION
**Pull request by mkoeppe**
### 📚 Description
Fixes #34957

### 📝 Checklist

- [x] I have made sure that the title is self-explanatory and the description concisely explains the PR.
- [x] I have linked an issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by mkoeppe** [comment]
Yes, that's an unrelated concurrency bug in ECL.

File: src/sage/stats/distributions/discrete_gaussian_polynomial.py [hunk 1]
```diff
@@ -54,7 +54,8 @@
 # policies, either expressed or implied, of the FreeBSD Project.
 #*****************************************************************************/
 
-from sage.rings.all import RR, ZZ
+from sage.rings.real_mpfr import RR
+from sage.rings.integer_ring import Z as ZZ
```
Comment (tobiasdiez) [inline src/sage/stats/distributions/discrete_gaussian_polynomial.py]:
```suggestion
from sage.rings.integer_ring import ZZ
```
?

**Review by tobiasdiez — DISMISSED** [review]

**Comment by tobiasdiez** [comment]
There are a few more:
```
from sage.rings.integer_ring import Z as ZZ
from sage.rings.rational_field import Q as QQ
```

**Comment by mkoeppe** [comment]
Thanks, I'll fix them with a script

**Review by dimpase — APPROVED** [review]
lgtm

**Comment by mkoeppe** [comment]
Thanks, Dima!

**Pull request merged**